src/components/grid/GridComponent.vue

```diff
@@ -2,74 +2,88 @@
   <div id="grid" class="variable-grid">
     <div class="row">
       <div class="col vld-parent grid-col">
-        <loading :active="isLoading" loader="dots" :is-full-page="false" color="var(--secondary)" background-color="var(--light)"></loading>
+        <loading
+          :active="isLoading"
+          loader="dots"
+          :is-full-page="false"
+          color="var(--secondary)"
+          background-color="var(--light)"
+        ></loading>
 
         <table
           ref="gridheader"
-          class="grid-table"
+          class="grid-header-table"
           :class="{'sticky':stickyTableHeader}">
           <tr>
             <th></th>
-            <td v-show="isSignedIn" ></td>
-            <td
+            <th></th>
+            <th
               v-for="assessment in gridAssessments"
               :key="assessment.id"
               class="text-center">
               <div class="assessments-title"><span>{{assessment.name}}</span></div>
-            </td>
+            </th>
           </tr>
         </table>
+
         <div :class="{'space-holder':stickyTableHeader}"></div>
 
-        <table ref="grid" class="grid-table col-hover" v-if="!isLoading && grid.length">
+        <table
+          v-if="!isLoading && grid.length"
+          ref="grid"
+          class="grid-table"
+          @click.stop="clickGridDelegate"
+          :class="{'hover-all-cells': hoverAllCells}">
           <tr>
             <th></th>
-            <td>
+            <th class="all-toggle">
               <button
-                id="grid-toggle-all-btn"
-                v-show="isSignedIn"
```

Comment:
If the use is not logged in. There should be no (disabled)buttons visible in the grid that can't be pressed.
The grid then is only a display of data.  ( Disabled buttons may add to confusion, and don't add value )

Reply:
Disabled grid-toggle buttons are now hidden by default